app/code/Magento/Sales/Model/ResourceModel/Report/Bestsellers/Collection.php

```diff
@@ -37,25 +49,39 @@ class Collection extends \Magento\Sales\Model\ResourceModel\Report\Collection\Ab
     ];
 
     /**
-     * @param \Magento\Framework\Data\Collection\EntityFactory $entityFactory
-     * @param \Psr\Log\LoggerInterface $logger
-     * @param \Magento\Framework\Data\Collection\Db\FetchStrategyInterface $fetchStrategy
-     * @param \Magento\Framework\Event\ManagerInterface $eventManager
-     * @param \Magento\Sales\Model\ResourceModel\Report $resource
-     * @param \Magento\Framework\DB\Adapter\AdapterInterface $connection
+     * @param EntityFactory          $entityFactory
+     * @param LoggerInterface        $logger
+     * @param FetchStrategyInterface $fetchStrategy
```

Comment:
Please revert the phpdoc elements align formatting changes

Reply:
fixed